File: dart/11d.dart
import 'dart:io';

void main() {
  String userAge = stdin.readLineSync()!;
  if (int.parse(userAge) >= 18) {
    print("You can see anything!");
  } else if (int.parse(userAge) >= 15 && int.parse(userAge) < 18) {
    print("You can see up to a 15");
  } else if (int.parse(userAge) >= 12 && int.parse(userAge)< 15 ){
    print("You can see upto a 12");
  } else {
    print("Your mam will collect you");
  }
}

File: c/11d.c
#include <stdio.h>
#include <stdlib.h>

int main() {
  int age;
  scanf("%d", &age);

  if (age < 12) {
    printf("You can watch that your mam will pick you up");
  } else if (age < 15) {
    printf("You can watch upto a 12");
  } else if (age < 18) {
    printf("You can watch upto a 15");
  } else {
    printf("You can watch upto an anything");
  }

  return 0;
}
